src/state/apis/nft/types.ts

```diff
@@ -0,0 +1,36 @@
+import type { AssetId, ChainId } from '@shapeshiftoss/caip'
+
+import type { MediaUrl } from '../zapper/validators'
+
+export type NftCollectionItem = {
+  // The AssetId of the collection, which doesn't exist as an assetSlice asset yet (https://github.com/shapeshift/web/pull/4555)
+  // Note, depending on the endpoint being hit, we may or may not be able to get an actual collection AssetId
+  id: AssetId | null
+  chainId: ChainId
+  // The name of the collection, not the name of the NFT
+  name: string
+  // Denominated in the native asset of the NFT chain
+  floorPrice: string | null
+  openseaId: string | null
+  description: string | null
```

Comment:
easier to just force string and set to `''` if no description for rendering purposes I would imagine

Reply:
Ehh why not

Reply:
b7215dee6b